2_adventures/001/alexsimmons1964/05/README.md

```diff
@@ -0,0 +1,92 @@
+So it's day 5 of this challenge and I haven't looked at any form of coding yet.
+
+I had hoped to try using someone else's code as a way to learn about how you do that e.g.
+download it, or copy it, or whatever the right terminology is, and somehow and make it
+run on my computer, but I am not sure how to go about that. Well not yet.
+ 
+So I asked the Gitterati (I just made that up) and of course got some helpful replies 
+which required use of commands I was not familiar with or understood.
+
+I looked up the Git documentation to read up on these commands, and of course the 
+documentation uses words and phrases in the definitions that I also don't understand.
+
+This pattern of assumed prior knowledge of terminology I'm finding to be a common theme 
+at this stage of my sizeable ignorance. I look up some of those and the references start 
+to become a bit circular. 
+
+Slowly, enough context is formed that a term starts to makes sense and I think I get it.
+
+Here's an example of what I mean. From the git documentation for git-push:
+
+	NAME
+	git-push - Update remote refs along with associated objects
+
+that's the first line of the definition. 
+
+In the context of my current level of knowledge, the only words in that line I probably 
+understand are "along with", and that's possibly a brave assumption on my part because 
+I'm assuming these are used in their regular English sense, and not some special 
+programming/git code sense. 
+
+IOW I'm assuming that each of:
+- Update
+- remote
+- refs
+- associated objects
+
+have a specific meanings in git land.
+
+It goes on:
+
+	DESCRIPTION
+	Updates remote refs using local refs, while sending objects necessary to complete the given refs.
```

Comment:
To be fair, it is not great that they use the 4 letters 'refs' as it really is just short for 'reference' and 'references' as that is what is going on.